src/001._two_sum.md

```diff
@@ -38,9 +38,8 @@ class Solution(object):
         for i in range(len(nums)):
             for j in range(i+1, len(nums)):
                 if nums[i] + nums[j] == target:
-                    return [i, j]
-                else:
-                    return None
+			return [i, j]
+        return None  #加上这个判断会超时
```

Comment:
If Time-Limit-Exceeded, then choose another way. The nested loops approach has time complexity O(n^2), which is indeed time-consuming. Therefore, the nested loops approach should be avoided whenever possible.

Reply:
OK， I will have a try

Reply:
@stormsunshine  I have tried to refine the code. Much thanks for reviewing the code again